### ISSUE TITLE
Update Directions link behavior on Vet Center pages

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#30918]
## Issue Description
This issue updates the Get Directions behavior, label, and announcements based on feedback during screen reader research and 508 standards. The changes have been made to Facility Locator search results and should be reflected across the product portfolio.  

---
## Tasks
- [ ] Update behavior when Veteran clicks on the Directions link
- [ ] Change "Get directions" label to "Directions on Google Maps"
- [ ] Change screen reader announcement to "Directions on Google Maps to" + name of location
- [ ]  use "aria-describedby" and/or "aria-label" where appropriate

## Acceptance Criteria
- [ ] When a Veteran clicks on the Directions link from a Vet Center page, Google Maps will open in the same tab.
- [ ] When a Veteran clicks on the back button from Google maps, users should be returned to the page from which the link was clicked.
- [ ] Link is labeled "Directions on Google maps"
- [ ] Screen reader announcement reads "Directions on Google Maps to" + name of location

**Comment by laflannery** [comment]
I reviewed multiple Vet Center pages and this looks resolved.  I will close this in 1 week unless told otherwise.
